docs/user_docs/kubeblocks-for-xinference/manage-xinference.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Manage Xinference with KubeBlocks
+description: How to manage Xinference on KubeBlocks
+keywords: [xinference, vector database, control plane]
```

Comment:
remove 'vector database' and keep consistent with xinference in api_docs

Reply:
Fixed.